src/components/policy/policy_external/include/policy/status.h

```diff
@@ -161,6 +162,37 @@ class UpdateNeededStatus : public Status {
   bool IsUpdateRequired() const OVERRIDE;
 };
 
+/**
+ * @brief The UpdatePendingStatus class represents 'update pending' status
```

Comment:
I think the description of the class should be more functional. The reasoning you gave in the PR description was really good. Maybe use some of that description here?

> The UpdatePendingStatus class represents cases when SDL knows that update is required and but before the snapshot is sent to the HMI.

Reply:
@JackLivio Done in 7d5222c6ca6e483113b51eae3332d9e94ec5444a